src/components/Table/helper.tsx

```diff
@@ -176,6 +177,27 @@ export const generateColumns = (columns: CustomColumn[]) => {
         }
         generatedColumns.push(el)
         return generatedColumns
+      case 'action':
+        el = {
+          Header: '',
+          accessor: element.accessor || '',
+          id: element.id,
+          className: element.className ? `${element.className} text-center` : 'text-center',
+          headerClassName: element.headerClassName || '',
+          disableSorting: true,
+          disableFiltering: true,
+          Cell: ({ row }: CellProps<object>) => (
+            <>
+              {element.id && (
+                <Button onClick={() => element.onClick(get(row.original, 'id', ''))}>
```

Comment:
The onClick function should just return the entire object the row is based on, rather than the `id`. 

This makes the code more flexible and allows the client to decide what the want to do with the data.